dictionaries/r/scripts/generate.r

```diff
@@ -0,0 +1 @@
+writeLines(ls(parent.env(globalenv())), file("../src/r.txt", "wt"))
```

Comment:
I think these words should be part of the file as well.

Reply:
Just checking, do you mean that there should be a `cspell:ignore` comment? If so, that's fixed in 0694979ad42828f3990657fe8692861e6ac3431c.

Reply:
Ok, so it looks like I didn't fully understand the way R handles environments and scopes. It looks like environments are layered on top of each other, and the package environments that are loaded by default are `baseenv()`, alongside whatever is hardcoded as the value of the `defaultPackages` option in `/usr/lib/R/base/R/Rprofile`. So I updated the generate script to capture all of those environments too in f13a13f7715fb5a04930581bd1a1ace5b952f3f7.